README.md

```diff
@@ -1,6 +1,6 @@
-# Rodot Engine
+# Redot Engine
 
-<p align="center">
+<p align="center">e
```

Comment:
...🤦‍♂️

Reply:
![2gdymy](https://github.com/user-attachments/assets/b197d45e-8164-40b4-acd6-3a10180cb4b9)

Reply:
🤪oh no, It seems like I accidentally touched the keyboard, I'm very sorry.